website/docs/sdk-reference/android.md

```diff
@@ -476,6 +476,17 @@ ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#", options -> {
     options.cache(new MyCustomCache()); // inject your custom cache
 });
 ```
+### Platform specific cache
```

Comment:
Could you please format this similar to the kotlin one, like:
```
## Cache
<Details about shared pref cache>

### Custom Cache
...
```

Reply:
Reworked.